kernel/mmanutils.c

```diff
@@ -664,6 +664,21 @@ static int _mmap_fd_checks(int prot, int flags, int fd)
     return ret;
 }
 
+int set_mem_usage(long length)
```

Comment:
Consider function names like `update_mem_usage` or `register_mem_usage`. This function is accounting for the memory usage update caused by the current allocation/deallocation.